core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -676,4 +678,13 @@ FlowIdentifier verticalScalingByName(
             @PathParam("name") String name,
             @QueryParam("initiatorUserCrn") String initiatorUserCrn,
             @Valid StackVerticalScaleV4Request updateRequest);
+
+    @GET
+    @Path("internal/used_subnets")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = GET_USED_SUBNETS_BY_ENVIRONMENT_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.STACK_NOTES,
+            nickname = "getUsedSubnetsByEnvironment")
+    UsedSubnetsByEnvironmentResponse getUsedSubnetsByEnvironment(
+            @PathParam("workspaceId") Long workspaceId,
+            @QueryParam("environmentCrn") String environmentCrn);
```

Comment:
could you add `ValidCrn` validation to these endpoints environmentcrn parameter?